utils/sub8_diagnostics/scripts/thruster_debug_shell.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ipython
+# flake8: noqa
```

Comment:
Why are we not format checking these files? I think there should always be a comment explaining why a file/line is ignored from flake8